examples/main.go

```diff
@@ -21,7 +21,11 @@ func main() {
 	projectID := "1234-56789-101112"
 	bucketName := "example"
 
-	err = c.ObjectStorage.Buckets.Create(context.TODO(), projectID, bucketName)
+	w, err := c.ObjectStorage.Buckets.Create(context.TODO(), projectID, bucketName)
```

Comment:
`if err != nil {...` should come before w.Wait is used
also, would be nice to name the variable `process` to match the usage in the terraform provider

```suggestion
	process, err := c.ObjectStorage.Buckets.Create(context.TODO(), projectID, bucketName)
	if err != nil {
		panic(err)
	}
	
	// wait for bucket to be created
	if err := process.Wait(); err != nil {
		panic(err)
	}
```